tools/scripts/sign-deploy-jar.sh

```diff
@@ -17,16 +17,24 @@
 # specific language governing permissions and limitations
 # under the License.
 
-# This is a general bash script to sign and deploy a datasketches-memory-X.jar.
-# This is intended to be used for releasing the Memory component to Maven central.
+# This is a general bash script to sign and deploy datasketches-memory-X.Y.Z.jars to Nexus.
+# This assumes that the jar files exist with X.Y.Z suffix in /target/ 
+#   and were created by "mvn clean install".
+# Also the project.basedir must be set to the X.Y.X candidate at the correct git hash
+# and there must be an X.Y.Z tag at that git hash.
+
+# Even for release candidates, e.g. "-RCn", it is recommended to deploy the candidates
+#   without the "-RCn". If the candidate succeeds, it is trivial just "release" them in Nexus.
+# If the candidate does not succeed, you can just "drop" them in Nexus for the next round.
 
 #  Required Input Parameters:
 #  \$1 = Git Version Tag for this deployment
-#       Example tag for SNAPSHOT         : 1.0.0-SNAPSHOT
-#       Example tag for Release Candidate: 1.0.0-RC1
-#       Example tag for Release          : 1.0.0
+#       Example tag for SNAPSHOT         : 1.0.0-SNAPSHOT  //not recommended
```

Comment:
Why do we list these forms if at all rather than just saying this should be done only for release tags?